data/json/items/book/electronics.json

```diff
@@ -168,6 +168,26 @@
     "time": "20 m",
     "fun": -1
   },
+  {
+    "id": "textbook_atomic",
+    "type": "BOOK",
+    "name": "Nuclear Physics Made Easy",
+    "name_plural": "Nuclear Physics Made Easy",
+    "description": "A book detailing the workings of state of the art atomic technology, and the physics behind it.  Many of the things detailed here would need some very advanced components and equipment to implement and therefore are practically useless now, however, more fundamental information presented in this book could indeed be useful.",
```

Comment:
I was going to say cut the references to craftables, but this has become incoherent, why is a book on nuclear physics teaching you up to an extremely high level of electronics?  Just remove the item.